docs/src/add-services/redis.md

```diff
@@ -4,16 +4,12 @@ weight: 11
 sidebarTitle: "Redis"
 ---
 
-Redis is a high-performance in-memory object store, well-suited for application level caching.
-
-See the [Redis documentation](https://redis.io/documentation) for more information.
-
+[Redis](https://redis.io/documentation) is a multi-model database that allows you to store data in memory
+for high-performance data retrieval and storage purposes.
 Platform.sh supports two different Redis configurations:
 
-- Persistent: useful for key-value application data
-- Ephemeral: in-memory only, useful for application caching
-
-Otherwise, they're identical.
+- [Ephemeral Redis](#ephemeral-redis): for application-level caching with no persistent storage
+- [Persistent Redis](#persistent-redis): for fast application-level key-value persistent storage
```

Comment:
:hammer_and_wrench: The double capitalization makes it look more like a proper noun. Maybe it wouldn't without both?
```suggestion
- [Ephemeral](#ephemeral-redis): for application-level caching with no persistent storage
- [Persistent](#persistent-redis): for fast application-level key-value persistent storage
```